### PR TITLE
Update secure-scan.yml for ESRP Malware Scan Azure Pipelines

### PR DESCRIPTION
**Pull request by kmyrup**
Adding in ESRP malware scanning task

### CODE REVIEW

File: pipelines/secure-scan.yml [hunk 1]
```diff
@@ -34,6 +34,17 @@ steps:
   - powershell: 'get-childitem env:'
     displayName: "Print variables"
 
+  - task: EsrpMalwareScanning@1
+    displayName: 'Run ESRP Malware Scan'
+    inputs:
+      ConnectedServiceName: 'ESRP Malware Scanning service connection'
+      FolderPath: '$(Build.SourcesDirectory)\objects'
```
Comment (silveyy) [inline pipelines/secure-scan.yml:41]: >objects [](start = 45, length = 7)

can we only scan the changed folder?

Comment (kmyrup) [reply pipelines/secure-scan.yml:41]: Until we can detect the changed folder reliably, I'd rather scan them all.

File: tools/ExtensionDownloader.ps1 [hunk 2]
```diff
@@ -164,13 +164,16 @@ function DownloadAndExpand
 
    if($null -ne $extensionJson)
     {
-        $extensionRootDirectory = (get-BuildRootDirectory); 
+        Write-Host "##vso[task.setvariable variable=codeSignEnabled]true"
+        
+        $repoRootFolder = (get-BuildRootDirectory);
+        $consoleExtensionFolder = $reporepoRootFolder + "\objects\ConsoleExtension\"
```
Comment (silveyy) [inline tools/ExtensionDownloader.ps1:170]: >repo [](start = 39, length = 4)

typo

**Review by silveyy — DISMISSED** [review]
:shipit:

**Review by silveyy — APPROVED** [review]
:shipit:

**Pull request merged**